sections/03_categorize.md

```diff
@@ -290,80 +290,95 @@ by allowing more efficient use of the costly practice of manual annotation.
+
+Several technological solutions have been proposed in this area. Solutions like
+DataShield [@doi:https://doi.org/10.1093/ije/dyu188] and ViPAR
+[@doi:10.1093/ije/dyv193] allows querying and combining of datasets and the
+calculation of summary statistics across sites without compromising patient
+identity. Continuous Analysis [@doi:10.1038/nbt.3780] can allow reproducible
+computing on private data. Using such techniques intermediate results can be
+automatically tracked and shared without sharing the original data.  which may
+help to address concerns around physician confidence. The EH4CR project
+[@doi:10.1016/j.jbi.2014.10.006] goes a step further by allowing secure EHR
+records to be queried remotely via an inter-mediation layer that translates
+local formats and datastores.  
+
 Even without sharing data, algorithms trained on confidential patient data may
 present security risks or accidentally allow for the exposure of individual
 level patient data. Tramer et al. [@arxiv:1609.02943] showed the ability to
 steal trained models via public APIs and Dwork and Roth
 [@doi:10.1561/0400000042] demonstrate the ability to expose individual level
-information from accurate answers in a machine learning model. Attackers can
-use similar attacks to find out if particular data instance was present in the
+information from accurate answers in a machine learning model. Attackers can use
+similar attacks to find out if particular data instance was present in the
 original training set for the machine learning model [@arxiv:1610.05820] - in
-this case, whether a person's record was present. There
-are solutions to this challenge. Training algorithms in a differentially private
-information from accurate answers in a machine learning model. There are
-solutions to this challenge. Training algorithms in a differentially private
-manner provides a limited guarantee that the algorithms output will be equally
-likely to occur regardless of the participation of any one individual. The limit
-is determined by a single parameter which provides a quantification of privacy.
-Simmons et al. [@doi:10.1016/j.cels.2016.04.013] present the ability to perform
-GWASs in a differentially private manner and Abadi et al. [@arxiv:1607.00133]
-show the ability to train deep learning classifiers under the differential
-privacy framework. Federated learning
-[@arxiv:1602.05629] and secure aggregations
+this case, whether a person's record was present. There are solutions to this
+challenge. Training algorithms in a differentially private information from
```

Comment:
Looks like some merge issues here that also already existed. I think that deleting the first `training algorithms ...` to the second `this challenge` will fix things. Do you mind doing this via your PR as well?